src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const page = document.querySelector('body');
+  const newElement = document.createElement('div');
```

Comment:
Not the best name for variable. Let's rename it
```suggestion
  const message = document.createElement('div');
```